AdyenCardScanner/Sources/Formatters/ExpireDateFormatter.swift

```diff
@@ -0,0 +1,51 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+protocol ExpireDateFormatting {
+    func date(from string: String) -> Date?
+}
+
+class ExpireDateFormatter: ExpireDateFormatting {
```

Comment:
I believe expireDate does not exist.

it's either expiryDate or expirationDate

Reply:
Yes, you're right. I read it so many times out there 😂